packages/manager/apps/web-domains/src/alldoms/pages/service/serviceList/serviceList.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import '@/alldoms/setupTests';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render } from '@testing-library/react';
+import React, { PropsWithChildren } from 'react';
+import { vi } from 'vitest';
+import ServiceList from './serviceList';
+
+vi.mock('react-router-dom', () => ({
+  useNavigate: () => vi.fn(() => null),
+  Navigate: vi.fn(() => null),
+  useLocation: vi.fn(() => ({
+    pathname: '',
+    search: '',
+  })),
+}));
+
+const queryClient = new QueryClient();
+const wrapper = ({ children }: PropsWithChildren) => (
+  <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
```

Comment:
same here, I'd do a "test provider" file and declare this in the file, to be able to reuse it